internal/config/cgmgr/cgmgr.go

```diff
@@ -72,6 +74,9 @@ type CgroupManager interface {
 	// CreateSandboxCgroup takes the sandbox parent, and sandbox ID.
 	// It creates a new cgroup for that sandbox, which is useful when spoofing an infra container.
 	CreateSandboxCgroup(sbParent, containerID string) error
+	// CreateSandboxCgroup takes the sandbox parent, and sandbox ID.
```

Comment:
typo: `s/CreateSandboxCgroup/RemoveSandboxCgroup/`.

Perhaps we should use a linter to catch this.  Revive would do (alas enabling it for the whole repo will cause tons of warnings)